Sources/Commands/SwiftTestCommand.swift

```diff
@@ -180,13 +180,13 @@ struct TestCommandOptions: ParsableArguments {
     var _testCaseSpecifier: String?
 
     @Option(help: """
-        Run test cases matching regular expression, Format: <test-target>.<test-case> \
-        or <test-target>.<test-case>/<test>
+        Run test cases that match a regular expression, Format: '<test-target>.<test-case>' \
+        or '<test-target>.<test-case>/<test>'.
         """)
     var filter: [String] = []
 
     @Option(name: .customLong("skip"),
-            help: "Skip test cases matching regular expression, Example: --skip PerformanceTests")
+            help: "Skip test cases that match regular expression, Example: '--skip PerformanceTests'.")
```

Comment:
nit: just to match the above help text

```suggestion
            help: "Skip test cases that match a regular expression, Example: '--skip PerformanceTests'.")
```